admin-client/src/main/java/com/epam/admin_client/service/ProjectService.java

```diff
@@ -0,0 +1,73 @@
+package com.epam.admin_client.service;
+
+import com.epam.admin_client.service.exception.ProjectException;
+import com.epam.admin_client.service.util.HeadersBuilder;
+import com.epam.test_generator.controllers.project.request.ProjectCreateDTO;
+import com.epam.test_generator.controllers.project.response.ProjectDTO;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import javax.annotation.Resource;
+import org.springframework.core.env.Environment;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpServerErrorException;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class ProjectService {
+
+    private final String URI_CREATE_PROJECT;
+    private final String URI_ADMIN_PROJECTS;
+
+    private RestTemplate restTemplate;
+    private ObjectMapper mapper;
+    private HeadersBuilder headersBuilder;
+    @Resource
+    private Environment environment;
+
+    public ProjectService(RestTemplate restTemplate, ObjectMapper mapper,
+                          HeadersBuilder headersBuilder, Environment environment) {
+        this.restTemplate = restTemplate;
+        this.mapper = mapper;
+        this.headersBuilder = headersBuilder;
+        this.environment = environment;
+        this.URI_CREATE_PROJECT = environment.getProperty("create.project");
+        this.URI_ADMIN_PROJECTS = environment.getProperty("admin.projects");
+    }
+
+    public ProjectDTO[] getAllProject(String token) throws IOException {
+        HttpEntity<String> entity = new HttpEntity<>(headersBuilder.tokenToHeaders(token));
+        ResponseEntity<ProjectDTO[]> result;
+        try {
+            result = restTemplate
+                .exchange(URI_ADMIN_PROJECTS, HttpMethod.GET, entity, ProjectDTO[].class);
+        } catch (HttpServerErrorException e) {
+            throw new ProjectException(e);
+        }
+        return result.getBody();
+    }
+
+    public void createProject(ProjectCreateDTO projectDTO, String token)
+        throws JsonProcessingException {
+        if (projectDTO != null) {
+            String jsonUserDTO = mapper.writeValueAsString(projectDTO);
+            HttpEntity<String> entity = new HttpEntity<>(jsonUserDTO,
```

Comment:
Can you use HttpEntity of ProjectCreateDTO instead of string? that will allow to remove object mapper